test/runner.c

```diff
@@ -160,6 +165,11 @@ int run_test(const char* test,
   main_proc = NULL;
   process_count = 0;
 
+#if defined(__ANDROID__)
+    /* Set fdsan to always warn on file descriptor misuse */
+    android_fdsan_set_error_level(ANDROID_FDSAN_ERROR_LEVEL_WARN_ALWAYS);
```

Comment:
Do we want to run this unconditionally? Maybe guard it with some ifdef to selectively run it?

Reply:
Thanks for the review!

> Do we want to run this unconditionally?

Do we? I can check that, for now, the current revision hangs in the emulator phase I'm fixing it

Reply:
I'll try to use the latests version of the android Image, so we could introduce more sanitizers if needed.